src/amo/api/index.js

```diff
@@ -99,35 +100,39 @@ type GetReviewsParams = {|
 type GetReviewsApiResponse = PaginatedApiResponse<ApiReviewType>;
 
 export function getReviews(
-  { user, addon, ...params }: GetReviewsParams = {}
+  { apiState, user, addon, ...params }: GetReviewsParams = {}
 ): Promise<GetReviewsApiResponse> {
   return new Promise((resolve) => {
     if (!user && !addon) {
       throw new Error('Either user or addon must be specified');
     }
     resolve(callApi({
+      // Make an authenticated request if an API token exists.
+      auth: Boolean(apiState && apiState.token),
+      state: apiState,
```

Comment:
Nitpick here but I find it easier to read key/value args like this when they're sorted alphabetically.